README.rst

```diff
@@ -251,31 +251,63 @@ Log watcher
 
 Configuration variables can be set via Env variables:
 
-- ``WATCHER_CONTAINERS_PATH``: Containers directory path mounted from the host (Default: ``/var/lib/docker/containers``)
-- ``WATCHER_AGENTS``: Comma separated string of required log processor agents. (Required. Example: "scalyr,appdynamics")
-- ``WATCHER_CLUSTER_ID``: Kubernetes Cluster ID.
-- ``WATCHER_KUBE_URL``: URL to API proxy service. Service is expected to handle authentication to the Kubernetes cluster. If set, then log-watcher will not use serviceaccount config.
-- ``WATCHER_KUBERNETES_UPDATE_CERTIFICATES``: [Deprecated] Call update-ca-certificates for Kubernetes service account ca.crt.
-- ``WATCHER_INTERVAL``: Polling interval (secs) for the watcher to detect containers changes. (Default: 60 sec)
-- ``WATCHER_DEBUG``: Verbose output. (Default: False)
+``WATCHER_CONTAINERS_PATH``
+  Containers directory path mounted from the host (Default: ``/var/lib/docker/containers``)
+
+``WATCHER_KUBERNETES_STRICT_LABELS``
```

Comment:
Can we standardise how to name these vars? We have them without "KUBERNETES" and "KUBE"...